modules/tracking/samples/lasot_benchmark.py

```diff
@@ -0,0 +1,188 @@
+import numpy as np
+import cv2 as cv
+import glob
+import argparse
+import os
+
+# Method used for evaluate corners coords 
+# Not used yet - need to re-write calculations with it
+def calc_coords(bb):
+    xmin = bb[0]
+    xmax = bb[0] + bb[2] - 1.0
+    ymin = bb[1]
+    ymax = bb[1] + bb[3] - 1.0
+    cx = bb[0] + (bb[2] + 1.0) / 2
+    cy = bb[1] + (bb[3] + 1.0) / 2
+    coords = {'xmin': xmin, 'xmax': xmax, 'ymin': ymin, 'ymax': ymax,
+                   'cx': cx, 'cy': cy}
+    return coords
+
+# Method used for initializing of trackers by creating it 
+# via cv.TrackerX_create()
+def init_tracker(tracker_name):
+    config = {"Boosting": (cv.TrackerBoosting_create(), 500),
+    "MIL": (cv.TrackerMIL_create(), 1000),
+    "KCF": (cv.TrackerKCF_create(), 110),
+    "MedianFlow": (cv.TrackerMedianFlow_create(), 20),
+    "GOTURN": (cv.TrackerGOTURN_create(), 20),
+    "MOSSE": (cv.TrackerMOSSE_create(), 20),
+    "CSRT": (cv.TrackerCSRT_create(), 250)}
+    return config[tracker_name]
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Run LaSOT-based benchmark for visual object trackers")
+    # As a default argument used name of 
+    # original dataset folder
+    parser.add_argument("--path_to_dataset", type=str,
+                        default="LaSOTTesting", help="Full path to LaSOT")
+    args = parser.parse_args()
+
+    # Creating list with names of videos via reading names from txt file
+
+    video_names = os.path.join(args.path_to_dataset, "testing_set.txt")
+    with open(video_names, 'rt') as f:
+        list_of_videos = f.read().rstrip('\n').split('\n')
+
+    trackers = [
+        'Boosting', 'MIL', 'KCF', 'MedianFlow', 'GOTURN', 'MOSSE', 'CSRT']
+
+    # Loop for every tracker
+    for tracker_id in range(len(trackers)):
+
+        tracker_name = trackers[tracker_id]
```

Comment:
Why not 

`for tracker_name in trackers:`

Reply:
OK.